Moodlet/app/src/androidTest/java/com/cmput3owo1/moodlet/activities/FilterTest.java

```diff
@@ -0,0 +1,90 @@
+package com.cmput3owo1.moodlet.activities;
+
+import androidx.test.espresso.contrib.RecyclerViewActions;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+
+import com.cmput3owo1.moodlet.R;
+import com.cmput3owo1.moodlet.activities.LoginActivity;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static androidx.test.espresso.Espresso.pressBack;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static com.cmput3owo1.moodlet.fragments.MoodHistoryFragmentTest.atPosition;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class FilterTest {
+
+    @Rule
+    public ActivityTestRule<LoginActivity> activityTestRule = new ActivityTestRule(LoginActivity.class, false, true);
+
+    // Start with MainActivity (JUnit Rules: https://developer.android.com/training/testing/junit-rules)
+    private String testEmail = "test@ua.ca";
+    private String testPassword = "123123";
+
+    private void loginWithTestAccount() throws InterruptedException {
+        // Logout first
+        openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getInstrumentation().getTargetContext());
```

Comment:
Does this cause an error if the user was never logged in? For us it won't because we've been logged in before, but if aaron runs it and he hasn't logged it maybe this line will cause a crash or something